indico/MaKaC/webinterface/tpls/ConferenceClone.tpl

```diff
@@ -1,5 +1,50 @@
 <script type="text/javascript">
-    var date = new Date();
+
+    function mod(x, m) {
```

Comment:
I think this function deserves an explanation. It's not really clear what it's supposed to do any the name `mod` is misleading since `mod` is just `x % m`